cloud-functions/comparer-function/main.py

```diff
@@ -32,7 +32,9 @@ def handle(request):
     previous_frame_path = download_blob(bucket_name, 'previous_frame.jpg')
     if previous_frame_path is not None:
 
-        percentage = compare(latest_frame_path, previous_frame_path)
+        image_mask = os.environ.get('IMAGE_MASK',None)
```

Comment:
check for `None` here instead of inside function.

Reply:
Added check for the Mask image